### DISCUSSION TITLE
Use Laravel Routes with Statamic

### DISCUSSION THREAD

**Discussion by iAbhinavJain** [statamic/cms#6327] [Q&A] [answered]
Hello to the amazing community, hope everyone is doing well. 

`Situation:`
- I have an existing database that I'll be using for generating some pages using Laravel (the traditional way, the DB has 'slug').
- I want to use it along with the Statamic routes I have set up a collection called 'post' and its dynamic slug is the same as the one I am using for DB (i.e. domain.tld/{slug}). 

`What I am trying to achieve:`
- For every matching route, I'd like to create a function that looks for slug in Statamic, if it matches, then it calls the standard statamic route and if not, it'll calls the standard laravel function which I'll be creating for the domain.tld{slug}. 

`P.S.: I am quite new to Statamic, so apologies if this is actually really easy. `

Thank you )

**Comment by jesseleite** [comment, marked as answer]
Totally possible with our query builder. You can read more about it here: https://statamic.dev/content-queries
